### PR TITLE
#480 Add '--dryrun' mode to the 'download sources' command

### LINKED ISSUE #480: Add '--dryrun' mode to the 'download sources' command
**Is your feature request related to a problem? Please describe.**

Some commands in the CLI have the `--dryrun` mode which allows printing a command output without execution. For example, the `crowdin upload sources` command.

This mode is missing in the `crowdin download sources` command and it would be great to have it here.

**Describe the solution you'd like**

Ability to run the `crowdin download sources --dryrun` without actually putting downloaded files on a local machine but seeing the command output in the terminal.

### PR DESCRIPTION
**Pull request by kiatt210**

### CODE REVIEW

File: src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java [hunk 1]
```diff
@@ -60,7 +59,7 @@ public void testDest() throws IOException {
         FilesInterface files = mock(FilesInterface.class);
 
         NewAction<PropertiesWithFiles, ProjectClient> action =
-            new DownloadSourcesAction(files, false, false, null, true, false);
+            new DownloadSourcesAction(files, false, false, null, true, false,false);
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java]:
```suggestion
            new DownloadSourcesAction(files, false, false, null, true, false, false);
```

File: src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java [hunk 2]
```diff
@@ -92,7 +91,7 @@ public void testDestAndUnaryAsterisk() throws IOException {
...
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java]:
```suggestion
            new DownloadSourcesAction(files, false, false, null, true, false, false);
```

File: src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java [hunk 3]
```diff
@@ -236,4 +235,30 @@ public void testWithPreserveHierarchyFalse() throws IOException {
         verify(files).writeToFile(eq(Utils.joinPaths(project.getBasePath(), "/android_4a.xml")), any());
         verifyNoMoreInteractions(files);
     }
+
+    @Test
+    public void testDryRun() {
+        PropertiesWithFiles pb = NewPropertiesWithFilesUtilBuilder
+                .minimalBuiltPropertiesBean(
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/cli/commands/actions/DownloadSourcesActionTest.java]: Please fix the indentations here and below

**Comment by andrii-bodnar** [comment]
Hi @kiatt210, Thanks a lot for your contribution! 🎉 

Please check the minor comments I've made to this PR

**Comment by kiatt210** [comment]
Hi @andrii-bodnar, I fixed the indentations.

**Comment by andrii-bodnar** [comment]
Hi @kiatt210, thanks, looks good to me!

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**